HelloWars/Arena/EliminationTypes/TournamentLadder/TournamentLadderControl.xaml.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+using Arena.EliminationTypes.TournamentLadder.UserControls;
+using Arena.Models;
+
+namespace Arena.EliminationTypes.TournamentLadder
+{
+    /// <summary>
+    /// Interaction logic for TournamentLadderControl.xaml
+    /// </summary>
+    public partial class TournamentLadderControl : UserControl
+    {
+        public TournamentLadderControl(List<Competitor> competitors)
+        {
+            InitializeComponent();
+            _competitors = competitors;
+        }
+
+        private List<Competitor> _competitors { get; set; }
+
+
+        private void TournamentLadderControl_OnLoaded(object sender, RoutedEventArgs e)
+        {
+            int competitorsNo = 16;
+
+            var roundNumbers = (int)Math.Ceiling(Math.Log(competitorsNo, 2));
+
+
+            Grid1.HorizontalAlignment = HorizontalAlignment.Left;
+            Grid1.VerticalAlignment = VerticalAlignment.Center;
+            Grid1.ShowGridLines = true;
+
+            for (int i = 0; i < roundNumbers; i++)
+            {
+
+                ColumnDefinition columndefinition = new ColumnDefinition();
+                Grid1.ColumnDefinitions.Add(columndefinition);
+
+                for (int c = 0; c < competitorsNo; c++)
```

Comment:
I'd avoid c for such variables; i stands for index, hence the common name; also - avoid abbreviations and be careful about English rules about plural usage: should be something like competitorCount (never ~~competitorsCount~~)